### DISCUSSION TITLE
M2M choose item multiple times

### DISCUSSION THREAD

**Discussion by ulvidamirli** [directus/directus#7873] [Archive]
I have an `M2M` relationship between `Posts` and `Tests` tables. I added additional columns to the `junction table` such as `test_results` and `targets`. So, I can add multiple IELTS test scores for different target users. To do that, I want to be able to choose the `Tests` item (in this case, IELTS) more than once and add different characteristics in the `junction table` but Directus doesn't allow me. 

Is there any way to be able to choose the item in `M2M` more than once?

**Comment by ulvidamirli** [comment]
I created a `junction table` myself. And it works as I want. How can I achieve the same thing on the default `M2M` interface that comes when creating `M2M` with Directus?

**Comment by rijkvanzanten** [comment]
> Is there any way to be able to choose the item in M2M more than once?

No, not at this time
